nemo_skills/dataset/utils.py

```diff
@@ -236,3 +236,38 @@ def get_mcq_fields(question, choices):
         "options": options_text,
         **options_dict,
     }
+
+
+def generate_subgroup_init_files(output_dir, subgroup_configs):
```

Comment:
please test if it works with just putting init.py as explicit files (not generating on the fly) and if so we can remove this new function